### ISSUE TITLE
Training dataset

### ISSUE THREAD

**Issue by pingpeng1017** [BreezeWhite/oemer#28]
I recently had the opportunity to explore your system, and I was impressed by its capabilities. However, I encountered an issue while trying to access the dataset used for training the first model. It appears that the dataset is currently unavailable on the provided page. If you happen to have the dataset, would it be possible for you to provide it? Thanks :)

**Comment by BreezeWhite** [comment]
Hi @pingpeng1017 , I think it's better to send an inquiry email to them and also let them know the dataset download page is down, since I am not sure if it's proper for me to directly share the data with you.

**Comment by pingpeng1017** [comment]
Thanks for your reply. I have discovered that the time signature is not being extracted and I'm curious whether it's not recognised at all, or if it's actually recognised but there is no implementation of the code to convert it to XML. I would greatly appreciate it if you could let me know the possibility of adding this functionality as I would like to have a go :)

**Comment by BreezeWhite** [comment]
The time signature does appear in the predictions of the two UNet models, but in raw pixel format. I did not managed to recognize the numbers of the time signature since it would take more efforts but would not impact the listening experience too much. Still you could try to recognize the symbol if it is important in your case.

**Comment by pingpeng1017** [comment]
Does that mean it's necessary to retrain the UNet models or as it's already appearing in the predicted images, would it be feasible to work with these raw pixel format images and change the SVM to recognise the time signature symbols? I'm currently working on a program that converts music notation into Braille, and being able to recognise time signature symbols would be a game-changer for my project. Any help or guidance you can provide would be incredibly valuable.

**Comment by BreezeWhite** [comment]
Ues, it's already in the predicted image. You only need to figure out a way to extract which pixels belong to the time signature and what numbers do they represent. For number recognition, you could train another SVM model to recognize, too.

**Comment by pingpeng1017** [comment]
So I can use the same DeepScores dataset for training the SVM model for number recognition right?

**Comment by pingpeng1017** [comment]
NOTE_TYPE_TO_RHYTHM = {
    NoteType.WHOLE: {"name": "whole", "duration": DIVISION_PER_QUATER * 4},
    NoteType.HALF: {"name": "half", "duration": DIVISION_PER_QUATER * 2},
    NoteType.**HALF_OR_WHOLE**: {"name": "half", "duration": DIVISION_PER_QUATER * 2},
    NoteType.QUARTER: {"name": "quarter", "duration": DIVISION_PER_QUATER},
    NoteType.EIGHTH: {"name": "eighth", "duration": DIVISION_PER_QUATER // 2},
    NoteType.SIXTEENTH: {"name": "16th", "duration": DIVISION_PER_QUATER // 4},
    NoteType.THIRTY_SECOND: {"name": "32nd", "duration": DIVISION_PER_QUATER // 8},
    NoteType.SIXTY_FOURTH: {"name": "64th", "duration": DIVISION_PER_QUATER // 16}
}

REST_TYPE_TO_DURATION = {
    RestType.QUARTER: DIVISION_PER_QUATER,
    RestType.EIGHTH: DIVISION_PER_QUATER // 2,
    RestType.SIXTEENTH: DIVISION_PER_QUATER // 4,
    RestType.THIRTY_SECOND: DIVISION_PER_QUATER // 8,
    RestType.SIXTY_FOURTH: DIVISION_PER_QUATER // 16,
    RestType.**WHOLE_HALF**: DIVISION_PER_QUATER * 2,
    RestType.WHOLE: DIVISION_PER_QUATER * 4,
    RestType.HALF: DIVISION_PER_QUATER * 2
}

What does the HALF_OR_WHOLE and WHOLE_HALF mean in this case?

**Comment by pingpeng1017** [comment]
Thank you so much! In the provided code, there is a list named CLASS_CHANNEL_LIST which groups various musical elements' colour channels together. Unfortunately, the specific meaning of each number for the time signatures is not explicitly provided in the code. I would like to know how to interpret these numbers and which time signature each number corresponds to. Could you please provide more information on how to understand the time signatures represented by the numbers in CLASS_CHANNEL_LIST? Sorry for too many questions 🥺

**Comment by pingpeng1017** [comment]
Hi, this is me again! Sorry for so many questions.
I've trained an SVM model to recognise numbers in the image and I remembered that you said I only need to find a way to extract pixels belonging to the time signature. It seems like you have implemented a method to extract pixels for three specific symbols below:

    stems_rests = np.where(sep==1, 1, 0)
    notehead = np.where(sep==2, 1, 0)
    clefs_keys = np.where(sep==3, 1, 0)
    
I'm curious about how you managed to obtain these values.
